### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Zesysy**

### CODE REVIEW

File: src/directory_home/Search-bar.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import { InputGroup, Button, FormControl, Dropdown, DropdownButton } from "react-bootstrap";
+import { Link, Redirect } from "react-router-dom";
+
+import styles from "./search.module.css";
+// import errorPopover from "./errorPopover";
```
Comment (Karnak19) [inline src/directory_home/Search-bar.jsx]: code mort

File: src/directory_home/Search-bar.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    handleChange2(event) {
+        this.setState({
+            inputValue: event.target.value
+        });
+    }
+
+    render() {
+        const { realmInput, inputValue, regionEU, regionUS } = this.state;
+
+        // if (inputValue === "") {
```
Comment (Karnak19) [inline src/directory_home/Search-bar.jsx]: same here

**Review by Karnak19 — CHANGES_REQUESTED** [review]
remove dead code and go :)

File: src/directory_home/Search-bar.jsx (hunk 2 above)
Comment (Zesysy) [reply src/directory_home/Search-bar.jsx]: do to commit eb5f49e

**Comment by EnzoDuh** [comment]
@Karnak19 Code's clean!

**Review by Karnak19 — APPROVED** [review]

**Pull request merged**